src/main/java/lrbg/codriver/command/EventCommand.java

```diff
@@ -0,0 +1,40 @@
+package lrbg.codriver.command;
+
+import lrbg.codriver.data.Event;
+import lrbg.codriver.data.exception.CoDriverException;
+import lrbg.codriver.data.Task;
+import lrbg.codriver.data.TaskList;
+import lrbg.codriver.storage.Storage;
+import lrbg.codriver.ui.Ui;
+
+import java.time.LocalDate;
+
+public class EventCommand extends Command {
+    private final String description;
+    private final LocalDate fromDate;
+    private final LocalDate toDate;
+
+    public EventCommand(String description, LocalDate fromDate, LocalDate toDate) {
+        this.description = description;
+        this.fromDate = fromDate;
+        this.toDate = toDate;
+    }
+
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws CoDriverException {
+        Task newTask = new Event(this.description, this.fromDate, this.toDate);
+        tasks.addTask(newTask);
+        ui.showAddTask(newTask, tasks.size());
+    }
+
+    public boolean testEquals(Object obj) {
```

Comment:
i wonder if testEquals is better placed somewhere else, like a CommandTest class built for testing instead?